{
  "file": "core/src/convert/num.rs",
  "name": "convert::num::<impl convert::TryFrom<i16> for u128>::try_from",
  "hash": "815295521860543473213507395237256255193",
  "hash_direct": "145590119325791245695309893889293346244",
  "src": "fn try_from(u: $source) -> Result<Self, Self::Error> {\n                if u >= 0 {\n                    Ok(u as Self)\n                } else {\n                    Err(TryFromIntError(()))\n                }\n            }",
  "macro_backtrace": [
    {
      "callsite": "impl_try_from_lower_bounded!(i16 => u16, u32, u64, u128)",
      "defsite": "macro_rules! impl_try_from_lower_bounded {\n    ($source:ty => $($target:ty),+) => {$(\n        #[stable(feature = \"try_from\", since = \"1.34.0\")]\n        #[rustc_const_unstable(feature = \"const_convert\", issue = \"143773\")]\n        impl const TryFrom<$source> for $target {\n            type Error = TryFromIntError;\n\n            /// Tries to create the target number type from a source\n            /// number type. This returns an error if the source value\n            /// is outside of the range of the target type.\n            #[inline]\n            fn try_from(u: $source) -> Result<Self, Self::Error> {\n                if u >= 0 {\n                    Ok(u as Self)\n                } else {\n                    Err(TryFromIntError(()))\n                }\n            }\n        }\n    )*}\n}"
    }
  ]
}
